source/tests/system/nirfmxinstr_driver_api_tests.cpp

```diff
@@ -102,6 +102,43 @@ nidevice_grpc::Session init_session(const client::StubPtr& stub, const std::stri
   return session;
 }
 
+::grpc::Status get_rfsa_session_array(const client::StubPtr& stub, const nidevice_grpc::Session& session, const std::vector<std::string>& session_names, GetNIRFSASessionArrayResponse& response)
+{
+  auto request = GetNIRFSASessionArrayRequest{};
+
+  for (const auto& name : session_names) {
+    request.add_session_names(name);
+  }
+  request.mutable_instrument()->CopyFrom(session);
+  ::grpc::ClientContext context;
+  // Note: this can't use the experimental::client APIs, because those omit the session name
```

Comment:
I considered adding session names to the client API. This is actually pretty easy to add, but does require that all init methods use it and all calls pass in something (empty string is fine). I decided against it though. That API is deliberately streamlined to make it easy to test common use cases. Adding session names wouldn't be a huge deal, but it does work against that.